sdcm/utils/benchmarks.py

```diff
@@ -129,9 +173,27 @@ def _get_number_of_cores_on_node(self) -> int:
         core_num = int(regex_pattern.search(cat_result).groupdict()["core_num"])
         return core_num
 
-    @staticmethod
-    def run_fio():
-        ...
+    def run_fio(self):
```

Comment:
Seeing this change I think that it must have been part of the first commit in this chain. 
It makes no sense to update the code you make in the same PR like this.

Reply:
It was hard not to update it, since these are the initial commits for this code. That being said, I did rebase the commits a bit to make it less of hassle to review the changes.